packages/pictograms/svg/3D chart.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
I don't think we can have the file name begin with a number, it looks like that's what's causing the CI errors